### PR TITLE
OSDOCS-2111 Add release note for Power of Two Random Choices balancing algorithm for Ingress Controller

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-2111

Preview: https://deploy-preview-33847--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-ingress-controller-power-of-two-random-choices

### CODE REVIEW

**Comment by bmcelvee** [comment]
Release note for https://github.com/openshift/openshift-docs/pull/33339.

**Comment by bmcelvee** [comment]
@Miciah would you mind giving this PR a review when you get a chance? Thank you!

**Comment by bmcelvee** [comment]
@quarterpin would you mind reviewing this PR, please? Thank you!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -453,6 +453,18 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
...
+[id="ocp-4-8-ingress-controller-power-of-two-random-choices"]
+==== Power of Two Random Choices balancing algorithm for Ingress Controller
+
+{product-title} Ingress Controllers now uses the link:https://www.haproxy.com/blog/power-of-two-load-balancing/[Power of Two Random Choices] balancing algorithm for more even balancing after reloading. The Power of Two Random Choices balancing algorithm is set by default by the `random` parameter for `haproxy.router.openshift.io/balance`. See xref:../networking/routes/route-configuration.adoc#nw-route-specific-annotations_route-configuration[Route-specific annotations] for more information.
+
+To revert the Ingress Controller back to use the Least Corrections balancing algorithm, you can use the following `oc patch` command to specify `leastconn` for `haproxy.router.openshift.io/balance`:
+
+[source,terminal]
+----
+$ oc -n openshift-ingress patch ingresses/default --type=merge --patch='{"spec":{"unsupportedConfigOverrides":"set-default-balancing-algorithm-to-leastconn"}}'
```
Comment (Miciah) [inline release_notes/ocp-4-8-release-notes.adoc:465]: The command needs to be updated:
```suggestion
$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge --patch='{"spec":{"unsupportedConfigOverrides":{"loadBalancingAlgorithm":"leastconn"}}}'
```
I realize it came from the enhancement document; sorry about that.  I'll update the enhancement with the correct command.

Comment (bmcelvee) [reply release_notes/ocp-4-8-release-notes.adoc:465]: Thank you!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -495,6 +495,18 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/using-an-egress-router-ovn.adoc[Considerations for the use of an egress router pod].
 
+[id="ocp-4-8-ingress-controller-power-of-two-random-choices"]
+==== Power of Two Random Choices balancing algorithm for Ingress Controller
+
+{product-title} Ingress Controllers now uses the link:https://www.haproxy.com/blog/power-of-two-load-balancing/[Power of Two Random Choices] balancing algorithm by default for more even balancing after reloading. As in earlier releases, the balancing algorithm for a specific route can still be configured using the `haproxy.router.openshift.io/balance` route annotation. See xref:../networking/routes/route-configuration.adoc#nw-route-specific-annotations_route-configuration[Route-specific annotations] for more information.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:501]: "Ingress Controllers now uses": subj-obj seems off here unless it's due to product name weirdness

Comment (jeana-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:501]: Could switch "more info sentence" to goal-first ("For more information, ....")

File: release_notes/ocp-4-8-release-notes.adoc [hunk 3]
```diff
@@ -495,6 +495,18 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
...
+
+To revert the Ingress Controller back to use the Least Corrections balancing algorithm, you can use the following `oc patch` command to specify `leastconn` as the default:
```
Comment (jeana-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:503]: could omit "you can"

**Review by jeana-redhat — COMMENTED** [review]
A question and a couple small wording considerations, LGTM

**Pull request merged**